src/libraries/System.Text.Json/Common/JsonNumberHandling.cs

```diff
@@ -29,6 +29,9 @@ enum JsonNumberHandling
 
         /// <summary>
         /// Numbers will be written as JSON strings (with quotes), not as JSON numbers.
+        /// <remarks>
+        /// This field is not defined by the JSON specification. Altering the default number handling can potentially produce JSON that cannot be parsed by other JSON implementations.
```

Comment:
nit, here and below.
```suggestion
        /// This behavior is not defined by the JSON specification. Altering the default number handling can potentially produce JSON that cannot be parsed by other JSON implementations.
```